### ISSUE TITLE
[Bug] CrashException: Ticking Particle

### ISSUE THREAD

**Issue by N08I40K** [MeteorDevelopment/meteor-client#1817]
### Describe the bug

I turned off the rendering of all particles in NoRender. Then I went to the hype.crubix.ru server (it happens only there). I stood for a couple of minutes in the hub and a crash occurred.
From the crash log, I realized that minecraft is trying to set a color for a particle that does not exist. 

Crash log: https://paste.ee/p/3gxt1.

P.S - It has been since 1.16.5, but I decided to write about it just now :)

### Steps to reproduce

```bash
Disable rendering of all particles in NoRender.
Go to the hype.crubix.ru server.
Wait a couple of minutes in the hub.
```

### Meteor Version

0.4.7. master (#1790) 

### Minecraft Version

1.18

### Operating System

Linux

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by sarnthilIsHorny** [comment]
same happens with a login portal of openanarchy.org

**Comment by MineGame159** [comment]
Fixed in f0ed1c50d1a951de4d79a95e5a3d31f178f33806.
